.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     - run: npm ci
     - run: npm run build
 
-    - uses: ros-tooling/setup-ros@0.0.16
+    - uses: mikaelarguedas/setup-ros@bump-colcon
```

Comment:
You'll need to revert that now :)

Reply:
yup! just wanted to get someone to witness it did pass CI before removing that commit ;)

Reply:
Commit removed